dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/task/AbstractTask.java

```diff
@@ -103,7 +103,6 @@ public AbstractTask<?> onlyIfAllFailed(@Nonnull Task<?>... tasks) {
    * @throws Exception If the task fails.
    */
   @CheckForNull
-  // @VisibleForTesting @ForOverride
```

Comment:
Can't say about `VisibleForTesting`, but I can see `ForOverride` being meaningful in this context. IMO it would be best to get info from the original authors what purpose they meant for some of the annotations in dwh-tools.

Do you know of any cases where a subclass needs to call `doRun` directly?

Reply:
I removed useless comment from the code. Do you suggest to improve  the code and add `@ForOverride` ?

Reply:
I know this is a tiny change, but the class is an API used by the entire `dumper.app` including parts not used by us and the annotation determines what is the correct usage of this API.

Reply:
It was commented for 2 years. 
There is a good practice to clean-up the code when you work with it in the scope for your tasks, otherwise the cost of support will be only growing.